sound/soc/sof/sof-pci-dev.c

```diff
@@ -111,6 +111,24 @@ static const struct sof_dev_desc cnl_desc = {
 };
 #endif
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_COFFEELAKE)
+static const struct sof_dev_desc cfl_desc = {
+	.machines		= snd_soc_acpi_intel_cnl_machines,
+	.resindex_lpe_base	= 0,
+	.resindex_pcicfg_base	= -1,
+	.resindex_imr_base	= -1,
+	.irqindex_host_ipc	= -1,
+	.resindex_dma_base	= -1,
+	.chip_info = &cnl_chip_info,
+	.default_fw_path = "intel/sof",
+	.default_tplg_path = "intel/sof-tplg",
+	.nocodec_fw_filename = "sof-cnl.ri",
+	.nocodec_tplg_filename = "sof-cnl-nocodec.tplg",
+	.ops = &sof_cnl_ops,
+	.arch_ops = &sof_xtensa_arch_ops
+};
```

Comment:
@plbossart cnl_desc and cfl_dsc look exactly the same. Any particular reason to keep them separate?

Reply:
yes, if we change the firmware name or apci machine list it's a one line change later on, vs. introducing an entire new descriptor.

Reply:
got it